Tasks/ServiceFabricUpdateAppVersions/Update-ApplicationVersions.ps1

```diff
@@ -29,10 +29,13 @@ try {
 
     $logAllChanges = ((Get-VstsInput -Name logAllChanges -Require) -eq "true")
 
-    $versionSuffix = Get-VstsInput -Name versionSuffix -Require
-    Write-Host (Get-VstsLocString -Key VersionSuffixLabel -ArgumentList $versionSuffix)
+    $replaceVersion = ((Get-VstsInput -Name versionBehavior -Require) -eq "Replace")
```

Comment:
was this one of the new inputs that was added after the first revision? if so the `-require` may cause an error if the definition was saved against a version of a task that didn't have this. i think the agent might overlay the defaults, so if this has a default in the task.json then it might not be an issue (i can't remember exactly how it works offhand). let me know if you need help investigating this further.